docs/account/user-session-management.md

```diff
@@ -0,0 +1,11 @@
+# User session management
+
+To reduce the risk of unauthorized access to your account, Codacy sets automatic expiration timeouts for every session. When a session expires, Codacy invalidates the session both on client and server side.
```

Comment:
@andrzej-janczak Can you please confirm if this is correct?
`When a session expires, Codacy **invalidates the session both on client and server side**.`

Reply:
Yes, this is what we want to achieve after execution of epic 👍